sandy/core/samples.py

```diff
@@ -132,60 +132,65 @@ def test_shapiro(self, size=None, pdf="normal"):
 
```

Comment:
I would consider adding a note on the random number generation in Fortran mentioning the behavior we encountered.

Reply:
The problem is actually that the docstring is incomplete. I will fix it.

Anyway, as I understand there is no issue with the random number generation in Fortran, but rather with the possibly different compilation options.
To me this is not related to the random number generation, hence, I won't comment on it.
Issues with the compilation can be retrieved from browsing this PR and do not need to be necessarily in the docstring.